app/webpack/stylesheets/_claims_importer.scss

```diff
@@ -11,7 +11,7 @@
   .js-enabled & {
     input[type="file"],
     .no-file-selected [type="submit"] {
-      @include visually-hidden;
+      @include govuk-visually-hidden;
```

Comment:
**yak-shaving-warning** but I do not believe we need this file at all as the import was removed

Reply:
Great, thank you. I will remove this.